scripts/AbinsModules/__init__.py

```diff
@@ -45,3 +45,8 @@
 from . import AbinsParameters
 from . import AbinsConstants
 from . import AbinsTestHelpers
+
+# set utf8 enconding for ABINS so that it can read German surnames from output DFT programs without any error
+import sys
```

Comment:
This will change the default encoding for absolutely everyone and it is generally discouraged to do this: http://stackoverflow.com/questions/3828723/why-should-we-not-use-sys-setdefaultencodingutf-8-in-a-py-script

Would the [`codecs.open`](https://docs.python.org/2/library/codecs.html#codecs.open) function be what we need here?